pynecone/utils/format.py

```diff
@@ -298,8 +298,7 @@ def format_event(event_spec: EventSpec) -> str:
     event_args = [
         wrap(format_event_handler(event_spec.handler), '"'),
     ]
-    if len(args) > 0:
```

Comment:
took this check out because specifying default values for `E` function on the client is prone to error due to how we create the `E` function call on the server.